### ISSUE TITLE
[MAS4.1.2] Name is not defined for close button.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#2052]
Actual Result:
User will unable to understand the purpose of the button.

Expected Result:
Name should be defined for the "Close" (x) button available under '
My Bots' section.

Test Environment
Platform: Mac 10.15.3 Beta (19D49f)
App: Bot Framework Emulator V4
App version: Version 4.7.0-99577 (4.7.0-99577.99577)
Tool: Voice Over

Repro Steps:
1.Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to controls (x) icon available under my bot section.
4. Observe the Voiceover announcement.

User Impact:
User will unable to understand the purpose of the control (X) button, if name is not properly defined it will impact of efficiency of the screen reader dependent user. 

Impacted MAS Link:
(https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

Attachment
[close button - DT issue.zip](https://github.com/microsoft/BotFramework-Emulator/files/4070621/close.button.-.DT.issue.zip)

**Comment by tonyanziano** [comment]
Ready to be reviewed in the latest nightly.

**Comment by ashish315** [comment]
@tonyanziano As we have checked the issue has been resolved. So, we are good to close the issue.
